source/general/faq.rst

```diff
@@ -32,3 +32,43 @@ We encourage members to make suggestions! Suggestions do not need to be based of
+
+Does ProjectKorra work for 1.16?
+================================
+
+The latest release, 1.8.9, was designed for Minecraft 1.15 and 1.14.
+If you installed it on your 1.16 server, the plugin would mostly work, but it would break under certain circumstances. Therefore, it's not recommended to use PK 1.8.9 for Minecraft 1.16.
+The next release will provide support for 1.16. The developers are working hard to ensure there are no game-breaking bugs before releasing, but the next update will come very soon and will add blue fire for firebenders, among other improvements.
+For now, you can try to compile the work-in-progress source code `here <https://projectkorra.com/forum/resources/projectkorra-core.1/>`_.
+
+How do I install ProjectKorra on my server?
+===========================================
+
+First, download ProjectKorra from the link above. Then, place the .jar file you downloaded into your server's "plugins" folder. Finally, restart your server.
+
+How do I install ProjectKorra on my Aternos or Minehut server?
+==============================================================
+
+Free Minecraft hosting websites come with limitations. Your site probably prevents you from installing a plugin the manual way, right into your "plugins" folder.
+If your hosting site has an automatic plugin installer, they probably detected the newest version as ProjectKorra 1.8.8 from the Spigot page, which unfortunately only works for Minecraft 1.13.
+We're currently unable to post an updated version on the Spigot page, which means you can't use your host site's plugin installer to install the newest version of ProjectKorra.
+For optimum customizability, we recommend the use of cheap (but not free) hosting websites or a server you host on your computer. There are many tutorials on how to set up a Minecraft server.
+
+How do I get the bending sideboard that shows my bound abilities?
+=================================================================
+
+ProjectKorra doesn't come with a bending sideboard. You'd have to install a side plugin for this.
+The well-known side plugins JedCore and ProjectAddons come with bending boards, but they also include a myriad of addon abilities.
+`JedCore download for Minecraft 1.15/1.14 <https://github.com/plushmonkey/JedCore/releases>`_
+`ProjectAddons download for Minecraft 1.15/1.14 <https://github.com/Simplicitee/ProjectAddons/releases>`_
+
+How do I use ProjectKorra on singleplayer?
+==========================================
+
+ProjectKorra is a Minecraft plugin, and plugins are intended for multiplayer servers.
+You can either try the Avatar 2 Mod, which is outdated and mostly unrelated to ProjectKorra, or you could make your own multiplayer server, install ProjectKorra, and just not portforward or tell others about it, basically making it a singleplayer experience.
```

Comment:
I don't think its outdated. The mod developers are consistently working on it, its just Forge development doesn't necesarily follow the same timeline as ours. I'd also include a link here just to be nice :)

Reply:
Ok, yeah sorry I didn't realize that.